Makefile

```diff
@@ -77,6 +77,12 @@ test-only:
 
 test: test-deps test-only
 
+integration-test: test-deps
+	@echo "=== $(INTEGRATION) === [ test ]: running integration tests..."
+	@docker-compose -f tests/integration/docker-compose.yml up -d
+	@go test -v -tags=integration ./tests/integration/. || (ret=$$?; docker-compose -f tests/integration/docker-compose.yml down && exit $$ret)
```

Comment:
The stuff behind the `||` makes sure the containers are freed even if the tests fail

Reply:
nice one 👍